app/themes/federal.tsx

```diff
@@ -163,18 +163,18 @@ theme.typography = merge(theme.typography, {
     fontWeight: "bold",
   }),
   body1: createTypographyVariant(theme, {
-    fontSize: [14, 16],
-    lineHeight: [22, 24],
+    fontSize: 14,
+    lineHeight: [24],
     fontWeight: "regular",
   }),
   body2: createTypographyVariant(theme, {
-    fontSize: [12, 14],
-    lineHeight: [18, 20],
+    fontSize: [14],
+    lineHeight: [20],
     fontWeight: "regular",
   }),
```

Comment:
Shouldn't body2 have different fontSize that body1? Or are the same in design specs?

Reply:
Extracted the commit, I will see this with @AnninaWalker